integration-tests/src/test/java/oracle/weblogic/kubernetes/ItJrfMiiDomain.java

```diff
@@ -213,33 +224,125 @@ public void testJrfModelInImage() {
         replicaCount,
         jrfMiiImage);
 
-    // create model in image domain
     createDomainAndVerify(domain, jrfDomainNamespace);
+    verifyDomainReady();
+  }
 
-    checkPodReadyAndServiceExists(adminServerPodName, domainUid, jrfDomainNamespace);
+  /**
+   * Save the OPSS key wallet from a running JRF domain's introspector configmap to a file
+   * Restore the OPSS key wallet file to a Kubernetes secret.
+   * Shutdown the domain.
+   * Using the same RCU restart the same JRF domain with restored OPSS key wallet file secret.
+   * Verify Pod is ready and service exists for both admin server and managed servers.
+   * Verify EM console is accessible.
+   */
+  @Order(2)
+  @Test
+  @DisplayName("Reuse same RCU to restart JRF domain")
```

Comment:
this name makes no sense.  RCU is a command line tool for setting up product schema in a database.  what are you actually reusing?  a set of schema?

Reply:
Yes. The same RCU schema. Let me change the method name as: 
@DisplayName("Reuse the same RCU schema to restart JRF domain")
  public void testReuseRCUschemalToRestartDomain() {